Gemfile

```diff
@@ -102,7 +102,8 @@ gem 'zk'
 gem 'mods', '~> 2.1'
 gem 'riiif', '~> 1.1'
 
-gem 'iiif_manifest', '~> 0.2.0'
+# gem 'iiif_manifest', '~> 0.2.0'
+gem 'iiif_manifest', git: 'https://github.com/ULCC/iiif_manifest', branch: 'hyku_1372_rendering'
```

Comment:
Yes, this PR is blocked until that happens. Just waiting for feedback on https://github.com/samvera-labs/iiif_manifest/pull/3